setup.cfg

```diff
@@ -2,11 +2,12 @@
 norecursedirs=node_modules bower_components
 xfail_strict=true
 
-[isort]
-line_length=80
-indent='    '
-multi_line_output=3
-known_third_party=notifications_utils,notifications_python_client
-known_first_party=app,tests
-include_trailing_comma=True
-use_parentheses=True
+[tool:isort]
+profile = black
+multi_line_output = 3
```

Comment:
I'm assuming the removal of some of the `isort` flags is because `black` will take care of those things going forward?

Reply:
yes this was the request from Steven, to eliminate conflicts between isort and black.  The black documentation suggests using the black profile, which eliminates the need for most of those other settings (when isort didn't have a black profile).

Reply:
black is very strict and opinionated and disagrees with how isort does something things by default.

Reply:
That makes two of us as far as `isort` is concerned... 😆 thank you!